### PR TITLE
Adds cadets, vulcans, andorians + new door sprites + morale officer + new circuitboard sprites

### PR DESCRIPTION
**Pull request by Kmc2000**
https://cdn.discordapp.com/attachments/360878113959182337/557580640824983552/unknown.png
https://cdn.discordapp.com/attachments/360878113959182337/557568270379515916/unknown.png

:cl: Kmc2000
add: You can now play as Vulcans and Andorians
add: The clown is now more lore friendly
add: Circuitboards were resprited to be isolinear chips
/:cl:

### CODE REVIEW

**Comment by spookydonut** [comment]
Atomise

File: DS13/goonstation/airlocks.dm [hunk 1]
```diff
@@ -19,7 +19,7 @@
 	req_one_access = list(ACCESS_SEC_DOORS)
 
 /obj/machinery/door/airlock/trek/goon/engi
-	name = "Engineering"
+	name = "Main Engineering"
```
Comment (Jalleo) [inline DS13/goonstation/airlocks.dm]: EW

**Review by Jalleo — CHANGES_REQUESTED** [review]
All engineering doors will be named main engineering.... Why isnt Engineering good enough?

**Review by Jalleo — APPROVED** [review]
Someone else can confirm its fine or just merge it.

**Pull request merged**